src/algorithms/tonal/chordsdescriptors.cpp

```diff
@@ -102,7 +102,8 @@ void ChordsDescriptors::compute() {
     throw EssentiaException("ChordsDescriptors: Chords input empty");
   }
 
-  string key = toUpper(_key.get());
+  string key = _key.get();
+  key[0] = toUpper(string(1, key[0]))[0];
```

Comment:
Would that fail on an empty key input? I guess we might have a unit test for that.

Reply:
Only the `chords` input was being checked. I added empty input checks for `key` and `scale` for consistency and unit tests for the 3 cases.